src/meshes/p4est_mesh.jl

```diff
@@ -323,10 +331,13 @@ For example, if a two-dimensional base mesh contains 25 elements then setting
 - `RealT::Type`: the type that should be used for coordinates.
 - `initial_refinement_level::Integer`: refine the mesh uniformly to this level before the simulation starts.
 - `unsaved_changes::Bool`: if set to `true`, the mesh will be saved to a mesh file.
+- `p4est_partition_allow_for_coarsening::Bool`: modify the partition in `p4est_partition` such that more
+                                                aggressive coarsening is possible in parallel setups using MPI.
```

Comment:
```suggestion
- `p4est_partition_allow_for_coarsening::Bool`: Must be `true` when using AMR to make mesh adaptivity
                                                independent of domain partitioning. Should be `false` for static meshes
                                                to permit more fine-grained partitioning.
```
Maybe more descriptive this way?

Reply:
Ah, the same suggestion for the other docstrings where applicable, of course.

Reply:
Done in 151d3bd